spec/administrate/views/fields/has_one/_form_spec.rb

```diff
@@ -2,21 +2,60 @@
 require "administrate/field/has_one"
 
 describe "fields/has_one/_form", type: :view do
-  it "displays the resource name" do
-    has_one = instance_double(
+  let!(:has_one) {
```

Comment:
[I would typically try and avoid using a `let`, let alone a `let!` as that might tell us something about the order dependency in the tests](https://thoughtbot.com/blog/lets-not).